packages/BUILD.bazel

```diff
@@ -2,5 +2,7 @@
 # targets so we can build them all at once
 filegroup(
   name = "build",
-  srcs = [],
+  srcs = [
+      "//packages/elastic-datemath:build"
```

Comment:
What purpose does `build` have instead of using `npm_module`?

Reply:
In that case is just an anchor to follow what we wrote on the RFC 😃

Reply:
👍 We can play around with it for whatever makes more sense. I am thinking that `npm_module` is more explicit in the context of the dependency you're requiring - where `build` doesn't tell much. Not needed now, but we might see that we will need to differentiate from dependencies that are being exported for consumption in server-side vs client-side vs types.

Reply:
Do you prefer we can just expose and use `npm_module` for now and omit `build`?

Reply:
I am not fully sure here - so let's just leave it for now.